Keas.Mvc/Keas.Mvc.csproj

```diff
@@ -5,6 +5,10 @@
    <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
     <TypeScriptToolsVersion>Latest</TypeScriptToolsVersion>  
   </PropertyGroup>
+  <ItemGroup>
+    <None Remove="ClientApp\components\Spaces\SpacesContainer.tsx" />
+    <None Remove="ClientApp\components\Spaces\SpacesListItem.tsx" />
+  </ItemGroup>
```

Comment:
@laholstege do these really need to be in the csproj?  Same with the later ones that are set to typescript compile.  why do we need them and just these?

Reply:
@srkirkland no vs added it and i forgot to remove it